x-pack/platform/plugins/shared/onechat/server/routes/chat.ts

```diff
@@ -60,9 +60,14 @@ export function registerChatRoutes({
         },
       })
     ),
-    input: schema.string({
-      meta: { description: 'The user input message to send to the agent.' },
-    }),
+    input: schema.maybe(
+      schema.string({
+        meta: { description: 'The user input message to send to the agent.' },
+      })
+    ),
+    confirm: schema.maybe(
+      schema.boolean({ meta: { description: 'Can be used to respond to a confirmation prompt.' } })
```

Comment:
I think this is too simple, we probably want something like a map of promptId -> promptResponse. We will probably revisit when implementing the UI